packages/react-composites/jest.config.js

```diff
@@ -8,10 +8,5 @@ const path = require('path');
 
 module.exports = {
   ...commonConfig,
-  roots: [
-    path.join(__dirname, 'src')
-    // Removing the `tests` dir from jest tests directories to prevent these tests
-    // from running with the command `npm test` | `rushx test`
-    // path.join(__dirname, 'tests')
-  ]
+  roots: [path.join(__dirname, 'src'), path.join(__dirname, 'tests')]
```

Comment:
Don't add these to CI just yet. I'm wary of running any tests in CI that depend on a real ACS backend service being available. CI should be hermetic.

Reply:
Yes, it's just for the draft to give everyone the complete picture of how the end product will look like :)

Reply:
I am curious if its possible to pass in a mock adapter where we can stub out the calls to the backend and have it return what we want to return for the puppeteer tests?

Reply:
A 100 times this!

My super-secret evil plan with https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/src/composites/ChatComposite/adapter/StubChatClient.ts is to implement a full fake for the backend services (i.e., fully functional implementation that's fully local and deterministic).

I think we absolutely need something like that to be able to run integration tests in CI.

Reply:
Can do but since we are using this for E2E tests, a stub would defeat the purpose. What do you think?

Reply:
The daydreaming idea isn't to implement stubs, but a fake -- a full implementation of the service backend, but hermetic (e.g, for chat this shouldn't be difficult -- just maintaining a few threads and users and sending notifications etc. in memory).

It makes sense for the service teams to maintain a fully-local fake implementation to be used for other services to run integration tests against. So really, the Calling team should maintain their fake etc.

But all this is highly tangential here.